setup.py

```diff
@@ -197,7 +209,18 @@ def run(self):
     setup_args['entry_points'] = {
         'console_scripts': [
             'jupyter-nbconvert = nbconvert.nbconvertapp:main',
-        ]
+        ],
+        "nbconvert.exporters" : [
+            'custom=nbconvert.exporters:TemplateExporter',
```

Comment:
Is `TemplateExporter` usable directly? How much extra info do you have to supply at the command line to use it?